my-app/src/components/championsGrid/ChampionsGrid.tsx

```diff
@@ -15,17 +14,59 @@ import {
 	sortAscending,
 	sortDescending,
 } from "../../features/champions/getChampions";
+import Champion from "../../features/champions/types/champion";
+import ToastContainerComponent from "../../components/toasts/toastContainer";
 
 const PaginatedItems = (props: any) => {
 	const dispatch = useDispatch();
 	const navigate = useNavigate();
 	const { id } = useParams();
 	const { page } = useParams();
 	const [show, setShow] = React.useState(false);
-	const [modalChampion, setModalChampion] = React.useState(0);
+	const [champion, setChampion] = React.useState<any>();
 	const reduxState = useSelector((state: any) => state.champions);
+	const watchlistState = useSelector((state: any) => state.watchlist);
 	const { loading, error, champions, totalPages } = reduxState;
```

Comment:
Looks like `reduxState` is only used once! How about destructuring directly from the `useSelector` output?

Reply:
Done